pom.xml

```diff
@@ -66,7 +66,7 @@
         <connection>scm:git:ssh://git@github.com/apache/druid.git</connection>
         <developerConnection>scm:git:ssh://git@github.com/apache/druid.git</developerConnection>
         <url>https://github.com/apache/druid.git</url>
-        <tag>0.19.0-SNAPSHOT</tag>
+        <tag>druid-28.0.0</tag>
```

Comment:
This seems odd. If we are rebasing on top of Druid 0.28 this should already be at 28 and not show 0.19.0-SNAPSHOT as the original tag

Reply:
In apache/druid, Tag is set to `0.19.0-SNAPSHOT` only. https://github.com/apache/druid/blob/254a8eb7e0c04eb8e6fe2c5d5950913faf821be2/pom.xml#L68C10-L69C35

Reply:
Created an upstream PR for the same: https://github.com/apache/druid/pull/15545